theme/themes/pastanaga/extras/objectbrowser-widget.less

```diff
@@ -30,4 +30,14 @@
 
 .ui.small.image.image-preview {
   margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.ui.input.file-widget-input {
+  display: none;
+}
+
+.label-file-widget-input {
+  padding: 5px;
+  border: 2px solid #ccc;
```

Comment:
Not a great fan of adding this in this file, since it should be dedicated to the object browser.

Reply:
Should I create a new file? what's your suggestion?

Reply:
Yes, I would create a separate file in order to separate concerns.